src/components/Input/Input.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import * as S from "./InputStyles";
+
+
+export const Input = (props) => {
+  const { label } = props;
+  return (
+    <div>
+      <S.Label>{label}</S.Label>
+      <S.Input type="text" />
```

Comment:
Acho que seria interessante esse type ser uma prop tbm

Reply:
verdade, vou alterar lá

Reply:
Opa, na verdade acho que podemos usar o spread aqui para jogar o que sobrar de props para o input.

[Dêem uma olhada](https://medium.com/coding-at-dawn/how-to-pass-all-props-to-a-child-component-in-react-bded9e38bb62) 👀